reconcile/ocm_addons.py

```diff
@@ -0,0 +1,88 @@
+import sys
+import logging
+
+import reconcile.queries as queries
+
+from reconcile.status import ExitCodes
+from utils.ocm import OCMMap
+
+QONTRACT_INTEGRATION = 'ocm-addons'
+
```

Comment:
Do we need a `QONTRACT_INTEGRATION_VERSION` here?

Reply:
the version is used for integrations that apply resources to OpenShift, so not really needed here.